suse_macros.in

```diff
@@ -13,8 +13,19 @@
 %make_install           make install DESTDIR=%{?buildroot}
 %makeinstall            make DESTDIR=%{?buildroot:%{buildroot}} install
 %insserv_prereq         insserv sed
-%fillup_prereq          fillup coreutils grep diffutils
+%fillup_prereq          fillup coreutils diffutils
 %install_info_prereq    info
```

Comment:
This should actually be dropped and we should use a file trigger in openSUSE instead.